daemon/api_debug.go

```diff
@@ -123,6 +123,14 @@ func postDebug(c *Command, r *http.Request, user *auth.UserState) Response {
 		return SyncResponse(true, nil)
 	case "get-base-declaration":
 		return getBaseDeclaration(st)
+	case "model":
```

Comment:
there was some misunderstanding, if it stays under  postDebug it needs to be "get-model" at this level, otherwise one wonder what's the action "model"